samples/gpu/multi.cpp

```diff
@@ -7,7 +7,7 @@
 #endif
 
 #include <iostream>
-#include "cvconfig.h"
+#include "opencv2/cvconfig.h"
```

Comment:
Actually if you plan to adapt this sample to make it working again, I suggest you completely remove old C-API stuff and replace it by the C++ variant.